### ISSUE TITLE
generated-flow-types doesn't report invalid imports

### ISSUE THREAD

**Issue by mrtnzlml** [relayjs/eslint-plugin-relay#59]
Hello, my colleague discovered that incorrect Relay imports are not reported as expected. Failing test (well, should be failing but it's not):

```js
{
  code: `
    import type {invalid as MyComponent_user} from './__generated__/MyComponent_user.graphql'
    class MyComponent extends React.Component<{user: MyComponent_user}> {
      render() {
        return <div />;
      }
    }

    createFragmentContainer(MyComponent, {
      user: graphql\`fragment MyComponent_user on User {id}\`,
    });
  `,
  errors: [
    {
      message: 'TODO: invalid type imported'
    }
  ]
},
```

Notice that the imported type `invalid as MyComponent_user` doesn't exist. In fact, this rule doesn't require the import at all and it is happy even if you delete the `import type` line completely. Seems like it takes into account only the property name used in props definition but not the imports. I believe this is not intended and it's a bug because seems like there are tests to cover this but they don't work as I would expect - the import is irrelevant for them: https://github.com/relayjs/eslint-plugin-relay/blob/267d5e64210b13ff733b104df938106d34fe14a0/test/test.js#L232-L233

cc @kassens, thanks for checking it and confirming/rejecting it
